editor/py/serve.py

```diff
@@ -490,72 +503,56 @@ def get_molfile():
         return 'no existing structural identifier', 404
 
 
-@app.before_first_request
-def init_submissions():
-    """Clones ord-data for use in review mode."""
-    # NOTE(kearnes): Use a local git clone so we aren't downloading the
-    # uncompressed submission files.
-    if not os.path.exists(app.config['REVIEW_DATA_ROOT']):
-        subprocess.run([
-            'git', 'clone',
-            'https://github.com/Open-Reaction-Database/ord-data.git',
-            app.config['REVIEW_DATA_ROOT']
-        ],
-                       check=True)
-
-
 @app.route('/review')
 def show_submissions():
-    """Lists pending submissions to ord-data."""
-    client = github.Github()
-    repo = client.get_repo('Open-Reaction-Database/ord-data')
-    pull_requests = {}
-    for pull_request in repo.get_pulls():
-        datasets = []
-        for file in pull_request.get_files():
-            if file.filename.endswith('.pbtxt'):
-                datasets.append(file.filename[:-6])
-        pull_requests[pull_request.number] = (pull_request.title, datasets)
+    """For the review user only, render datasets with GitHub metadata."""
+    if flask.g.user_id != REVIEWER:
```

Comment:
Maybe navigating to /review should automatically log in as the review user? Requiring the extra login step when the intention is clear seems inappropriate. However, this does give users a chance to record their user ID before logging out so they don't lose access to their work.

I think what I want here is for users to be able to enter the review portal without having to log out. Thinking ahead, when we switch to GitHub authentication we can't require users to log into a reviewer account; instead, we just want to show them the pending submissions in read-only mode and let them easily get back to their personal datasets.

I'm happy to punt on this and allow it to be addressed in a follow-up PR.

Reply:
Agreed there are UX and permission design questions here. Let's follow up.